Content.Server/Singularity/EntitySystems/RadiationCollectorSystem.cs

```diff
@@ -36,7 +56,32 @@ private void OnInteractHand(EntityUid uid, RadiationCollectorComponent component
 
         private void OnRadiation(EntityUid uid, RadiationCollectorComponent component, OnIrradiatedEvent args)
         {
-            if (!component.Enabled) return;
+            if (!component.Enabled || component.RadiationReactiveGases == null)
+                return;
+
+            if (!TryGetLoadedGasTank(uid, out var gasTankComponent) || gasTankComponent == null)
```

Comment:
Isn't the `gasTankComponent == null` check redundant? Or is the nullability checker not smart enough to silence a nullability warning without this?

Reply:
Unfortunately it is not :(